app/models/shipit/merge_request_assignment.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Shipit
+  class MergeRequestAssignment < Record
```

Comment:
Is this dead code? I can't see it used anywhere?

Reply:
Aye, good catch. It looks like its removal was missed in the merge that renamed the model-formerly-known-as-PullRequest to MergeRequest. Putting together a commit to remove it now.

Reply:
I removed it in powerhome#76